### PR TITLE
do not assign to same user again

### PR DESCRIPTION
**Pull request by mrfinch**
Also, how to run the unit tests for this engine?

I tried by cloning this code in discourse and running specs there. Is there a better way to do it?
Shouldn't tests be run on PR in this repo too?

### CODE REVIEW

File: spec/requests/assign_controller_spec.rb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  context 'assign' do
+
+    it 'assigns topic to a user' do
+      sign_in(user)
+
+      put '/assign/assign', params: {
+        topic_id: post.topic_id, username: user2.username
+      }
+
+      expect(response.status).to eq(200)
```
Comment (SamSaffron) [inline spec/requests/assign_controller_spec.rb]: minor but worth a followup PR I would recommend testing that the topic actually got assigned, if we are already doing all the work of assigning the topic.

**Comment by SamSaffron** [comment]
Thanks! looks good.

File: spec/requests/assign_controller_spec.rb (hunk 1 above)
Comment (mrfinch) [reply spec/requests/assign_controller_spec.rb]: hey, is there any better way to run unit-tests for this directly here? 

Also, I wanted my message to be i18n supported? Should I first create an entry in discourse in en.yml file and then add a corresponding entry here?

Comment (SamSaffron) [reply spec/requests/assign_controller_spec.rb]: do you mind heading to https://review.discourse.org/t/fix-do-not-assign-to-same-when-already-assigned/574 I will put the feedback there ... I want to mark it for followup anyway

**Pull request merged**